assets/css/styles.css

```diff
@@ -184,3 +184,18 @@
     overflow-y: auto;
     height: 500px;
 }
+
```

Comment:
Can you please explain why these changes were made? It seems that its related to your specific theme and the plugin, or preferred style.

Preferred style or issue specific to your theme would be better removed from the PR. The change might affect other users who are using the plugin with different theme.

I test the plugin with Moodle standard bootstrap theme. Users are always free to change the style in their own instance :)